ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -1,2298 +1,2328 @@
+    #             testunit_name, testplans_testunits_names_in_popup[0]['test_units'][0]))
+    #         self.assertEqual(testunit_name, testplans_testunits_names_in_popup[0]['test_units'][0])
+    #
+    #     def test061_testplans_popup_after_edit_by_add(self):
+    #         """
+    #         Orders: Test plan pop up  Approach: Make sure In case you edit the test plans
+    #         & add another ones this update should reflect on the test plan pop up
+    #         LIMS-8256
+    #         """
+    #         self.info('Get completed test plan to upade by it with raw material type')
+    #         testplan = \
+    #             self.test_plan_api.get_completed_testplans_with_material_and_same_article(material_type='Raw Material',
+    #                                                                                       article='', articleNo='')[0]
+    #         order, payload = self.orders_api.create_new_order(materialTypeId=1)
+    #         self.info('open the order record in the edit mode')
+    #         self.orders_page.get_order_edit_page_by_id(id=order['order']['mainOrderId'])
+    #         self.info('go to update the test plan by adding the completed one')
+    #         self.order_page.update_suborder(sub_order_index=0, test_plans=[testplan['testPlanName']])
+    #         self.order_page.save(save_btn='order:save')
+    #         testplan_name = testplan['testPlanName']
+    #         self.info("Get the test plan name that I added it in the edit mode".format(testplan_name))
+    #         testplans_testunits_names_in_popup = self.order_page.get_testplan_pop_up()
+    #         testunit_no = self.test_plan_api.get_testplan_with_quicksearch(quickSearchText=testplan_name)[0]['number']
+    #         testunit_name = self.test_plan_api.get_testunits_in_testplan_by_No(no=testunit_no)[0]
+    #         self.info("assert that the test plan I added in the test plan popup ".format(testplan_name,
+    #                                                                                      testplans_testunits_names_in_popup[
+    #                                                                                          1]['test_plan']))
+    #         self.assertEqual(testplan_name, testplans_testunits_names_in_popup[1]['test_plan'])
+    #         self.info("assert that the test plan I added in the test plan popup ".format(testplan_name,
+    #                                                                                      testplans_testunits_names_in_popup[
+    #                                                                                          1]['test_units'][0]))
+    #         self.assertEqual(testunit_name, testplans_testunits_names_in_popup[1]['test_units'][0])
+    #
+    #     @skip("https://modeso.atlassian.net/browse/LIMSA-127")
+    #     def test062_testplans_popup_after_edit_by_replace(self):
+    #         """
+    #         Orders: Test plan: Test 
```

Comment:
you don't used this, so no need to get it

Reply:
done